Sources/Filters+Strings+Boolean.swift

```diff
@@ -0,0 +1,54 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import Stencil
+
+extension Filters {
+  enum Strings {
```

Comment:
I've done it in 73ca5da. But there is one thing worth to mention. The order of compile sources must be like that, so `Filters+Strings.swift` is compiled first 😄 
<img src="https://monosnap.com/file/7dMO5BESY1Wvbaa5qO5rqU1sM4sHbe.png">